app/src/main-process/menu/menu-event.ts

```diff
@@ -16,7 +16,7 @@ export type MenuEvent =
   | 'show-preferences'
   | 'choose-repository'
   | 'open-working-directory'
-  | 'update-branch'
+  | 'update-branch-with-contribution-target-branch'
```

Comment:
Theses id's seems to be duplicated in `app/src/models/menu-ids.ts`. Just curious if there's a reason for this.